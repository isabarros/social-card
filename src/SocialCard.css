.user-picture {
  width: 100px;
  height: auto;
  border-radius: 50%;
}