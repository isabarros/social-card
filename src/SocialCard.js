import React from 'react';
import './SocialCard.css';

const SocialCard = () => {
    return(
        <div>
            <img className='user-picture'
                 src='https://res.cloudinary.com/practicaldev/image/fetch/s--nBBZ2m5n--/c_limit%2Cf_auto%2Cfl_progressive%2Cq_auto%2Cw_880/https://thepracticaldev.s3.amazonaws.com/i/ls565bghkq9uv8icae8d.png'
                 alt='user'/>
            <h3>Deva</h3>
            <p>Hello, world!</p>
        </div>);
};

export default SocialCard;