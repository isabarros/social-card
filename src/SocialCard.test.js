import React from 'react';
import SocialCard from './SocialCard';
import renderer from 'react-test-renderer';

it('renders properly', () => {
  const socialCard = renderer.create(<SocialCard/>);
  expect(socialCard).toMatchSnapshot();
});