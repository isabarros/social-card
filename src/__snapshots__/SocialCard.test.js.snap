// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders properly 1`] = `
<div>
  <img
    alt="user"
    className="user-picture"
    src="https://res.cloudinary.com/practicaldev/image/fetch/s--nBBZ2m5n--/c_limit%2Cf_auto%2Cfl_progressive%2Cq_auto%2Cw_880/https://thepracticaldev.s3.amazonaws.com/i/ls565bghkq9uv8icae8d.png"
  />
  <h3>
    Deva
  </h3>
  <p>
    Hello, world!
  </p>
</div>
`;
